jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/python/SamInitDirectoryBasedSettingsPanel.java

```diff
@@ -44,8 +46,9 @@ public JPanel getComponent() {
 
     @NotNull
     public ValidationResult validate() {
-        if (samExecutableField.getText().isEmpty()) {
-            return new ValidationResult(message("lambda.run_configuration.sam.not_specified"));
+        String validationMessage = SamCommon.Companion.validate(StringUtil.nullize(samExecutableField.getText()));
+        if (validationMessage != null) {
+            return new ValidationResult(validationMessage);
```

Comment:
The PyCharm warning label is not HTML by default.

![image](https://user-images.githubusercontent.com/742829/48747900-db307380-ec29-11e8-953a-c2adfad713d9.png)

Reply:
Using `<html>` tag fixed the problem.